### ISSUE TITLE
Ledger wallet electron app?

### ISSUE THREAD

**Issue by ghost** [LedgerHQ/ledger-nano-s#17]
Hello 
when will you offer Ledger wallet electron app?

**Comment by jonathancross** [comment]
Please explain better what you are looking for... "electron" can mean many things.  Do you mean this kind of "electron app" https://electron.atom.io ?  If you mean this "[electron coin](https://coinmarketcap.com/currencies/electron/)" -- it seems quite dead and is extremly unlikely to be added.

**Comment by ghost** [comment]
Yes, https://electron.atom.io/ when are you building a electron app for
ledger wallet(Bitcoin/Ethereum/Ripple combined) or use the brave browser
instead of using the google app? When will you be open myetherwallet on
brave browser with ledger nano S/Blue?

**Comment by jonathancross** [comment]
@briannyeko Unless there is a reason specifically for it to be built with Electron, I'd suggest you file two new issues with these titles:
1. **[feature request] Firefox support** - Can mention that you want to use with myetherwallet and that Chrome Apps wille be deprecated in early 2018: https://blog.chromium.org/2016/08/from-chrome-apps-to-web.html
2. **[feature request] Native app (eg Electron?)** - Can mention the deprecation of Chrome Apps link above.

Then close this issue.
You can also try adding here: https://trello.com/b/5nQ1mdzt/ledger-roadmap

**Comment by kaliumxyz** [comment]
With the release of [ledger live](https://github.com/LedgerHQ/ledger-live-desktop) this issue can be closed.
